src/main/java/no/digipost/signature/client/core/internal/ClientHelper.java

```diff
@@ -96,30 +92,31 @@ public void cancel(Cancellable cancellable) {
             String url = cancellable.getCancellationUrl().getUrl();
             Response response = postEmptyEntity(url);
             Status status = Status.fromStatusCode(response.getStatus());
-            if (status == CONFLICT) {
+            if (status == OK) {
+                return;
+            } else if (status == CONFLICT) {
                 throw new NotCancellableException();
             }
-            handleGeneralError(response, status);
+            throw handleGeneralError(response, status);
         } else {
             throw new NotCancellableException();
```

Comment:
In which cases does a Cancellable _not_ have a cancellationUrl? (is it still "cancellable" ;) )

Is it possible to add some more info to the NotCancellableException in case of status == CONFLICT? As it stands now, it is not possible to distinguish if the cancel()-method was erroneously called for an entity without any cancellationUrl to post to, or if the request was made but the server declined the cancellation for some reason.